stable/admin/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
Not sure what the implications are of automatically creating a role/SA, so I conditionalized it. Just about everything else seems to be conditionalized by some variable in the values files, so I figured I might as well follow that pattern.

With that said, it might be a good idea to reduce the number of permutations of parameters and define more things "by convention".